### DISCUSSION TITLE
Early reviews/feedback testers wanted!

### DISCUSSION THREAD

**Discussion by gabestein** [knowledgefutures/pubpub#2194] [Support]
Hey folks — we're about ready to test out a very early version of our new reviews feature (which we may rename "feedback"). Based on early conversations with many of you, the first iteration is very, very simple, relying on a special link that exposes a basic feedback form that can be filled out by a logged-in or logged-out user. Over time, we plan to add invitations, anonymization options, structured feedback forms...and of course your ideas, too!

Tantalizing sneak peak below. If you're interested in testing, reply here with your community URL or send a note to hello@pubpub.org and we'll send you the instructions.

Thanks, as always, for all of your help making PubPub better!

<img width="1082" alt="Screen Shot 2022-08-18 at 16 11 57" src="https://user-images.githubusercontent.com/639110/185486630-eb9762aa-6e1c-4f71-95c3-f2ca5e151818.png">

**Comment by CASCS** [comment]
We would like to test the new reviews feature.

journalofstrengthandperformance.com
